ui/src/data/virtual_school.js

```diff
@@ -135,4 +167,212 @@ export const allStudents = [
+    "behavior": "Sent to principal for fighting with CJ",
+    "ell": null,
+    "learningDisabilities": null,
+    "interests": "In drama club and loves art",
+    "familyBackground": "Single mom, two older sisters",
+    "ses": null,
+    "academicPerformance": "C in 6th grade science, B in 6th grade science"
+  }
+].map(addFields({
+  isMursion: false,
+  cohortKey: '20160722'
+}));
+
+
+// From Virtual School Context_Update_072716
+const julyStudentsFromMursion = [
+  {
+    "id": 3001,
+    "grade": "7th grade",
+    "name": "Maria Gonzales",
+    "gender": "female",
+    "race": "Dominican",
+    "behavior": "Quiet",
+    "ell": null,
+    "learningDisabilities": null,
+    "interests": "Loves to draw, write, and read",
+    "familyBackground": "Lives with both parents, 2 older brothers",
+    "ses": null,
+    "academicPerformance": "A in 6th grade science, A in 6th grade math"
+  }, {
+    "id": 3002,
+    "grade": "7th grade",
+    "name": "Cindy “CJ” Harper",
+    "gender": "female",
+    "race": "Caucasian",
+    "behavior": "Sent to principal for fighting with Lucy",
+    "ell": null,
+    "learningDisabilities": null,
+    "interests": "Loves movies, spends lots of time with 9th grade boyfriend",
+    "familyBackground": "Mom is ill, Dad is not present, older brother is often truant from high school",
+    "ses": null,
+    "academicPerformance": "F in 6th grade science, C in 6th grade math"
+  }, {
+    "id": 3003,
+    "grade": "7th grade",
+    "name": "Kevin Jordan",
+    "gender": "male",
+    "race": "African American",
+    "behavior": null,
+    "ell": null,
+    "learningDisabilities": null,
+    "interests": "Plays guitar, loves mixing music",
+    "familyBackground": "Lives with grandmother, ½ sister lives with Mom and boyfriend",
+    "ses": "Free and reduced lunch",
+    "academicPerformance": "C in 6th grade science, C in 6th grade math"
+  }, {
+    "id": 3004,
+    "grade": "7th grade",
+    "name": "Ed Lewis",
+    "gender": "male",
+    "race": "African American",
+    "behavior": null,
+    "ell": null,
+    "learningDisabilities": null,
+    "interests": "Wants to be an air force pilot like uncle, plays basketball",
+    "familyBackground": "Lives with both parents, younger sister and younger twin brothers",
+    "ses": null,
+    "academicPerformance": "B in 6th grade science, B in 6th grade math"
+  }, {
+    "id": 3005,
+    "grade": "7th grade",
+    "name": "Sean McGowen",
+    "gender": "male",
+    "race": "Caucasian",
+    "behavior": "Talkative",
+    "ell": null,
+    "learningDisabilities": null,
+    "interests": "Boy scouts, video games, math camp",
+    "familyBackground": "Lives with both parents, only child",
+    "ses": null,
+    "academicPerformance": "B in 6th grade science, C in 6th grade math"
+  }
+].map(addFields({
+  isMursion: true,
+  cohortKey: '20160722'
+}));
+
+export const allStudents = initialStudents.concat(
+  julyStudentsFromMursion,
+  julyStudentsNotFromMursion
+);
```

Comment:
Yeah, this is a great point!  In general, I think the idea is that the list of students is generally immutable, and that rather than changing students we create new variants of them.  This helps make clear that we look back at data from 7/12 that says "this person said X for student Y" that the meaning of student Y doesn't change after the fact.

In https://github.com/mit-teaching-systems-lab/threeflows/pull/105/files#diff-72bb72db1d42868cb18fa1c57dbf268eR21 it filters for only the latest cohort, so to make it simpler for #104 I moved that into `virtual_school.js` so that you can pull from activeStudents as well and get the desired autocomplete.

Thanks for the great feedback!